source/welcome/insights.rst

```diff
@@ -13,7 +13,7 @@ From Mattermost v7.1, insights are available for all users using Mattermost in a
   
 .. note:: 
     
-   To disable Insights, system admins can set the feature flag ``MM_FEATUREFLAGS_INSIGHTSENABLED`` to ``false`` to disable it server-wide. It can't be disabled on a per-user basis. The code is located `here <https://github.com/mattermost/mattermost-server/blob/master/model/feature_flags.go#L11>`_.
+   To disable Insights, system admins can set the feature flag ``MM_FEATUREFLAGS_INSIGHTSENABLED`` to ``false`` to disable it server-wide. It can't be disabled on a per-user basis. See the Knowledge Base article,  `How to disable Insights in Mattermost <https://support.mattermost.com/hc/en-us/articles/13897774815636-How-to-disable-Insights-in-Mattermost>`_, for details.
```

Comment:
We can link to this docs section from the changelog and upgrade notes:
 - https://docs.mattermost.com/upgrade/important-upgrade-notes.html (last bullet point)
 - https://docs.mattermost.com/install/self-managed-changelog.html#known-issues (second bullet point)

This can either be done in this PR, or I'm happy to make the changes in a separate PR.

Reply:
Hi @amyblais, I can do that. The link to code in the last bullet point of the Important Upgrade Notes is broken. Should I still include it?

Also, I may delete and resubmit this PR because of the issue with the other commits it has included. I did not properly base my branch off of the 7.11 branch.

Reply:
Thank you!
Thank you for noticing the broken link - the correct link is https://github.com/mattermost/mattermost-server/blob/master/server/model/feature_flags.go#L11.